examples/flava/native/utils.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import random
+from typing import Any
+
+import torch
+from hydra.utils import instantiate
+from omegaconf import DictConfig, OmegaConf
+from torch import distributed as dist
+
+# optional syntax-highlighting for console output
+try:
+    from rich.console import Console
+
+    c = Console(force_terminal=True)
+    print = c.log
+except ImportError:
+    pass
+
+
+def build_config() -> DictConfig:
```

Comment:
cant u just use the existing build config method?

Reply:
are you referring to the one in examples/flava/utils.py? it has some lightning specific settings